src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProviderTests.cs

```diff
@@ -130,5 +130,27 @@ class Bar : I2
             await VerifyItemExistsAsync(markup, "Goo2()");
             await VerifyItemExistsAsync(markup, "Prop");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
```

Comment:
There needs to be a `WorkItem` attribute linking to the issue this is fixing. You should be able to find examples either in this file or in other test classes by searching for "WorkItem".